tests/nightly/models/performance_thresholds.yaml

```diff
@@ -114,7 +114,7 @@ stfpm:
     image_AUROC: 0.857
     pixel_AUROC: 0.962
     nncf:
-      image_AUROC: 0.979
+      image_AUROC: 0.902
```

Comment:
In the future we could maybe replace this with an average value and ± tolerance. But for now this should work.

Reply:
Yes! This is an ideal approach for papers too